### ISSUE TITLE
Plasmamen never ever stop jittering

### ISSUE THREAD

**Issue by 9600bauds** [d3athrow/vgstation13#8231]
Their jitteriness var is always set to 30 and never goes down even after being warmed up.

**Comment by ihadtoregisterforthis** [comment]
Mild hypothermia starting at as little as a snowball hit.

**Comment by ghost** [comment]
Oversight, not my fault some eejit didn't allow plasmamen to jitter.
I'll fix this tomorrow.

**Comment by 9600bauds** [comment]
>not my fault some eejit didn't allow plasmamen to jitter.
????
Was this the case for anything that modified the jitteriness var, then?

**Comment by ghost** [comment]
Presumably. I didn't touch plasmamen jitter code, and if it's not ticking down there's no other explanation.

**Comment by ghost** [comment]
If so, reword this to talk about all forms of jitteriness.

**Comment by 9600bauds** [comment]
Does this still happen

**Comment by clusterfack** [comment]
Yes apparently it occurs because handle_chemicals_in_body has

```
	if(species.name=="Plasmaman")
		return 0 //Plasmaman shouldn't be hurt by plasma contaminated clothes
```

really early on in the proc, preventing them from reaching points where things like jitteriness are reduced.

And no, I don't think this was a good idea. Whoever did that was a fucking idiot.
